lib/filestreamer.js

```diff
@@ -205,7 +205,11 @@ FileStreamer.prototype._putFile = function (start, callback) {
  */
 FileStreamer.prototype._closeFile = function () {
 	if (this._fd) {
-		fs_module.close(this._fd);
+		fs_module.close(this._fd, (close_err) => {
```

Comment:
Can you rewrite this without an arrow function? We don't have any other ES6 stuff in this library (yet).